Team_47/QuizCreator/WebContent/CSS/instruction.css

```diff
@@ -78,46 +78,47 @@ a {
 }
 
 .rainbow-button {
-  width:calc(10vw + 6px);
-  height:calc(5vw + 6px);
-  background-image: linear-gradient(90deg, #00C0FF 0%, #FFCF00 49%, #FC4F4F 80%, #00C0FF 100%);
-  border-radius:5px;
-  display:flex;
-  align-items:center;
-  justify-content:center;
-  font-size:3vw;
-  font-weight:bold;
-  margin-top: 10px;
+	width: calc(10vw + 6px);
+	height: calc(5vw + 6px);
+	background-image: linear-gradient(90deg, #00C0FF 0%, #FFCF00 49%, #FC4F4F 80%, #00C0FF
+		100%);
+	border-radius: 5px;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	font-size: 3vw;
+	font-weight: bold;
+	margin-top: 10px;
 }
+
 .rainbow-button:after {
-  content:attr(alt);
-  width:10vw;
-  height:5vw;
-  background-color:#191919;
-  display:flex;
-  align-items:center;
-  justify-content:center;
+	content: attr(alt);
+	width: 10vw;
+	height: 5vw;
+	background-color: #191919;
+	display: flex;
+	align-items: center;
+	justify-content: center;
 }
+
 .rainbow-button:hover {
-  animation:slidebg 2s linear infinite;
+	animation: slidebg 2s linear infinite;
 }
 
-@keyframes slidebg {
-  to {
-    background-position:20vw;
-  }
+@

```

Comment:
This looks uglier than before. Was this intentional?

Reply:
Made by auto format in Eclipse. Done fixing them.